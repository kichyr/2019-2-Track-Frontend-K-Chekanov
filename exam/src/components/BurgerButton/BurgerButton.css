
.burgerButt  {
    justify-content: center;
    flex: 0.32;
    height: 100%;
    display: flex;
    align-items: center;
    flex-direction: column;
    transition: background-color 0.2s;
}
.burgerButt:hover, .burgerButtContainer:active {
    background-color: #1231FB;
}

.burgerButt > div{
    width: 2.5em;
    height: 0.5em;
    background-color: white;
    margin: 0.1em 3em;
    border-radius: 5%;
}