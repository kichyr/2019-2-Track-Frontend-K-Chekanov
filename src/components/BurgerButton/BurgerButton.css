
.burgerButt  {
    justify-content: center;
    flex: 0.32;
    height: 100%;
    display: flex;
    align-items: center;
    flex-direction: column;
    transition: background-color 0.2s;
}
.burgerButt:hover, .burgerButtContainer:active {
    background-color: #1231FB;
}

.burgerButt > div{
    width: 3em;
    height: 0.6em;
    background-color: white;
    margin: 0.1em 3em;
    border-radius: 10%;
}