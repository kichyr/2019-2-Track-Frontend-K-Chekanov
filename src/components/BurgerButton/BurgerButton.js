import React from 'react'
import './BurgerButton.css'

function BurgerButton() {
  return (
    <div className="burgerButt">
      <div />
      <div />
      <div />
    </div>
  )
}
export default BurgerButton
