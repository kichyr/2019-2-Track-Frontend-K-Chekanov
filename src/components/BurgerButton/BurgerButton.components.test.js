import React from 'react'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

import BurgerButton from './BurgerButton'

jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    push: jest.fn(),
  }),
}))

it('renders with or without props', () => {
  const output = shallow(<BurgerButton />)
  expect(output).toMatchSnapshot()
})
