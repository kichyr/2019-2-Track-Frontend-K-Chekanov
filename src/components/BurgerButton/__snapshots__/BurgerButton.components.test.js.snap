// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with or without props 1`] = `ShallowWrapper {}`;
